test/unit/filters/timeFilterSpec.js

```diff
@@ -0,0 +1,67 @@
+/* global describe, it, inject, expect, beforeEach, Config */
+
+describe('time filter', function () {
+  var $filter
```

Comment:
Today I've learned that you can share variables by assigning them to `this`.

See above for explanation.